doc/source/contribution/documentation/introduction.rst

```diff
@@ -6,6 +6,19 @@ Introduction
 This documentation describes about how to write, generate and manage
 Groonga documentation.
 
+How to clone Groonga repository
+-------------------------------
+
+First, please fork Groonga repository on GitHub because
+you need to build Groonga documentation from the latest source code.
+You just access https://github.com/groonga/groonga and press
+`Fork` button. Now you can clone your Groonga repository.
+
+.. code-block:: console
+
+  % git clone https://github.com/${YOUR_GITHUB_ACCOUNT}/groonga.git
```

Comment:
Is this indent correct?

```suggestion
   % git clone https://github.com/${YOUR_GITHUB_ACCOUNT}/groonga.git
```

Reply:
I think it looks fine judging from https://raw.githubusercontent.com/otegami/groonga/improve-how-to-contribute-in-docs/doc/source/contribution/documentation/introduction.rst.

Reply:
What is judged?

It seems that `c` position in `code-block` and `%` position don't have the same indentation.

Reply:
Ahh, I'm so sorry. I understood what you mentioned mm

Reply:
fix: [b663156](https://github.com/groonga/groonga/pull/1712/commits/b66315623f1a71ef84fc7e5deedf41f60640cc40) I thought the indentation wasn't correct before only `git clone ...` command.